docs/src/pages/guides/deploy.md

```diff
@@ -335,6 +335,18 @@ Use the following build settings:
 
 Then click the **Save and Deploy** button.
 
+## Render
+
+You can deploy your Astro project on [render](https://render.com/) following these steps:
+
+1. Create a [render.com account](https://dashboard.render.com/) and sign in
+2. Click the "**new+**"-button at the dashboard and select "**Static Site**"
+3. Connect your [GitHub](https://github.com/) or [GitLab](https://about.gitlab.com/) repository or alternatively enter the public URL of a public repository
+4. Give your website a name (1. to find it on the dashboard. 2. The onrender.com url is prefixed with it.), select the branch and specify the build command and publish directory  
+   - **build command:** `npm run build`
+   - **publish directory:** dist
```

Comment:
```suggestion
   - **publish directory:** `dist`
```

Consistency with previous line

Reply:
My intension was to make a difference between command and folder name.

Reply:
That makes sense to me. But the rest of the document does style folder names as code right now. If we change that we should change all of them (outside the scope of this PR).

Reply:
I agree. In the context that both is filled in in a form it make sense to have both code styled.